packages/lib/modules/pool/LbpDetail/LbpDetail.tsx

```diff
@@ -10,7 +10,7 @@ import { GqlPoolLiquidityBootstrappingV3 } from '@repo/lib/shared/services/api/g
 import { now } from '@repo/lib/shared/utils/time'
 import { isAfter, isBefore, secondsToMilliseconds } from 'date-fns'
 import { Top10Holdings } from './Top10Holdings'
-import { LbpPoolChartsContainer } from './LbpPoolChartsContainer'
+import { LbpPoolChartsContainer } from './LbpPoolCharts/LbpPoolChartsContainer'
```

Comment:
Why use Kebab / Pascal case for folders? Isn't the kebab case the convention for naming folders in node and most of the frameworks? (it has something to do with OS issues around casing and URL conventions)

Reply:
both are used so i guess we need to decide what to use only